pkg/minishift/registration/utils.go

```diff
@@ -30,7 +31,7 @@ type RegistrationParameters struct {
 }
 
 // Register host VM
-func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+func RegisterHostVM(host *host.Host, param *RegistrationParameters, proxyInfo string) error {
```

Comment:
No, what I am suggesting is independent of where we put the options. All global (credential + proxy settings, all per sub-command or a mix which we have now).

Whether or not we want to address issue #432 right now we can discuss.